recipes/geotrans/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
+
+find_package(geotrans REQUIRED COMPONENTS dtcc ccs)
+#
+set(EXTERNAL_LIBS 
+    geotrans::dtcc
```

Comment:
Are both components needed?
Can one component be used without the other?

Reply:
Yep, both components are needed to run the provided sample code. My understanding is that `geotrans::dtcc` can be used independently of `geotrans::ccs`, but `geotrans::ccs` requires `geotrans::dtcc` to run.